xml/art_migration.xml

```diff
@@ -1512,7 +1512,16 @@ fi</screen>
     <para>
      If your environment includes &rhla; &compnode;s, prepare your deployer for
      provisioning &ostack; services on them by following the steps described in
-     <xref linkend="install_rhel_compute_node"/>.
+     <xref linkend="install_rhel_compute_node"/>. If you install the SMT server
+     on your new deployer for mirroring the <literal>CentOS</literal> repo, note
+     that you must first run the command
+     <command>sudo ln -s /var/lib/mysql/mysql.sock /var/run/mysql/mysql.sock</command>.
+     Note also that after using the CA Management procedure to set up a root
```

Comment:
Suggested rephrase: "You must run <command>sudo ln -s /etc/ssl/ca-bundle.pem /etc/ssl/certs/ca-certificates.crt</command> after using the CA Management procedure to set up a root CA certificate to restore normal functionality to the node." (also removes the "note" repetition)